tools/cldr-code/src/main/java/org/unicode/cldr/util/VoterInfoList.java

```diff
@@ -0,0 +1,209 @@
+package org.unicode.cldr.util;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeMap;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.unicode.cldr.util.VoteResolver.Level;
+import org.unicode.cldr.util.VoteResolver.VoterInfo;
+
+public class VoterInfoList {
```

Comment:
this new class is a huge improvement for separation of concerns!

let's consider similar treatment for VoteResolver.Level and VoteResolver.VoterInfo